base/namedtuple.jl

```diff
@@ -103,6 +103,8 @@ firstindex(t::NamedTuple) = 1
 lastindex(t::NamedTuple) = nfields(t)
 getindex(t::NamedTuple, i::Int) = getfield(t, i)
 getindex(t::NamedTuple, i::Symbol) = getfield(t, i)
+getindex(t::NamedTuple, v::AbstractVector{Symbol}) = [t[i] for i in v]
```

Comment:
I think this is wrong and/or surprising. I'd do it as
```suggestion
getindex(t::NamedTuple, v::AbstractVector{Symbol}) = (; map(i->i => t[i], t)...)
```

Reply:
In particular, indexing a tuple with a vector gives you back a tuple, so by analogy, indexing a named tuple with a vector should give you back a named tuple.